libusb/os/windows_common.c

```diff
@@ -565,6 +566,20 @@ static int windows_init(struct libusb_context *ctx)
 
 	r = LIBUSB_SUCCESS;
 
+	if (init_count == 1) {
+		r = windows_start_event_monitor();
+	}
+	if (r != LIBUSB_SUCCESS) {
+		usbi_err(ctx, "error starting hotplug event monitor");
+
```

Comment:
If init_count wouldn't be 1, we couldn't arrive here, right? So why the second if clause below?

Reply:
Correct, there is a problem here, probably leftover from various tries, thanks. I'll fix this by simplifying.